packages/web-runtime/src/pages/account.vue

```diff
@@ -77,6 +77,7 @@
         <dd data-testid="language">
           <oc-select
             v-if="languageOptions"
+            :placeholder="$gettext('Please choose...')"
```

Comment:
We need to include translations in the PR to `stable-7.0`. Let's do the transifex-dance quickly...

Reply:
I started a translation sync just now... next step: translate in transifex, then again translation sync, then cherry-pick the commit into this PR

Reply:
where does it land ? core ?

Reply:
all done.